virtme_ng/run.py

```diff
@@ -1186,7 +1189,7 @@ def _get_virtme_append(self, args):
         append = []
         if args.append is not None:
             for item in args.append:
-                split_items = item.split()
+                split_items = shlex.split(item)
                 for split_item in split_items:
                     append.append("-a " + split_item)
```

Comment:
Can you please change the whole function.... it looks very suspicious to me.

```
        append = []
        if args.append is not None:
            for item in args.append:
                split_items = shlex.split(item)
                for split_item in split_items:
                   append += ["-a", split_item]
        if args.debug:
            append + = ["-a", "nokaslr"]
        self.virtme_param["append"] = shlex.join(append)
```

Still not beautiful and untested.

Reply:
And might be a good idea to use the long option for better readability (although `--kopt` reads not that good :D).

Reply:
Good idea! Will do!